test/e2e/framework/util.go

```diff
@@ -3216,33 +3216,6 @@ func UpdateDeploymentWithRetries(c clientset.Interface, namespace, name string,
 	return deployment, err
 }
 
-// Prints the histogram of the events and returns the number of bad events.
-func BadEvents(events []*api.Event) int {
```

Comment:
This was used to only detect whether we have some pod failures. We are checking the same now in the main function too.